pkg/plugins/platform/utils.go

```diff
@@ -23,7 +23,7 @@ func GetApps() map[string][]App {
 	apps := make(map[string][]App)
 	for platformType := range PluginMap {
 		labels := map[string]string{
-			"com.ddev.platform":          platformType,
+			"com.ddev.platform":          "ddev",
```

Comment:
drud legacy/dev uses the container name prefix to determine its listings, so I don't think it's actually necessary to change the plugin name to avoid conflict there. The vast majority of our usage for labels was introduced after we split off ddev, previous to that we relied heavily (almost entirely) on the name of the container. It doesn't harm anything to change plugin name though, so I'm good either way.